tools/buildsys/src/builder.rs

```diff
@@ -38,6 +38,9 @@ static DOCKER_BUILD_FRONTEND_ERROR: &str = concat!(
     r#"frontend grpc server closed unexpectedly"#
 );
 
+/* We also see sporadic CI failures with only this error message. */
+static UNEXPECTED_EOF_ERROR: &str = "unexpected EOF";
```

Comment:
I can live with this given the pain it's caused, but this seems like too common a phrase for me to be totally comfortable claiming that any build with this output can be retried.  We could be ignoring real errors, and if they're not consistent, we might miss them entirely.

One option to be a little more careful would be to modify buildsys to check for the error messages taking an entire line of output, not just appearing anywhere in the output.

Reply:
I did an alternate solution that lets us be a little more careful about error message positioning: https://github.com/bottlerocket-os/bottlerocket/pull/1557